developer_manual/client_apis/OCS/index.rst

```diff
@@ -0,0 +1,67 @@
+.. _webdavindex:
+
+=========
+OCS API's
+=========
+
+This document provides a quick overview of the OCS API endpoints supported in Nextcloud.
+
+All requests need to provide authentication information, either as a Basic Auth header or by passing a set of valid session cookies, if not stated otherwise.
+
+
+----
+Testing requests with curl
+----
+
+All OCS requests can be easily tested out using :code:`curl` by specifying the request method (:code:`GET`, :code:`PUT`, etc) and setting a request body where needed.
+
+For example: you can perform a :code:`GET` request to get information about a user:
+
+
+.. code-block:: bash
+
+    curl -u username:password -X GET 'https://cloud.example.com/ocs/v1.php/...' -H "OCS-APIRequest: true"
+
+
+-------------
+User metadata
+-------------
+
+Since: 11.0.2, 12.0.0
```

Comment:
Is it worth mentioning, that since 9 the endpoint is available and outputs a subset of this?
```xml
  <quota>
   <free>25956700160</free>
   <used>164</used>
   <total>25956700324</total>
   <relative>0</relative>
   <quota>-3</quota>
  </quota>
  <email/>
  <displayname>User One</displayname>
```